File: Basic-Video-Chat-With-ForegroundServices-Java/app/src/main/java/com/tokbox/sample/basicvideochatwithforegroundservices/network/GetSessionResponse.java
package com.tokbox.sample.basicvideochatwithforegroundservices.network;

import com.squareup.moshi.Json;

public class GetSessionResponse {
    @Json(name = "apiKey") public String apiKey;
    @Json(name = "sessionId") public String sessionId;
    @Json(name = "token") public String token;
}

File: E2EE-Video-Chat-Kotlin/app/src/main/java/com/tokbox/sample/ee2evideo/network/GetSessionResponse.kt
package com.tokbox.sample.ee2evideo.network

import com.squareup.moshi.Json

class GetSessionResponse {

    @Json(name = "apiKey")
    var apiKey: String = ""

    @Json(name = "sessionId")
    var sessionId: String = ""

    @Json(name = "token")
    var token: String = ""

    @Json(name = "secret")
    var secret: String = ""
}